### DISCUSSION TITLE
Failed to detect language used in a repository

### DISCUSSION THREAD

**Discussion by selcukemiravci** [github-linguist/linguist#5630] [Classification] [answered]
I have a repository called [PersonalWebsite](https://github.com/emiravc/PersonalWebsite). It is primarily written in HTML,CSS and some Javascript. But Linguist library failed to detect my repository is written in which language. It doesn't show any language in repository home page.

**Comment by Nixinova** [comment, marked as answer]
The reason is because your files are in a folder called "docs", and ↓
https://github.com/github/linguist/blob/4ce6df6a920f9e507d349da374453d0b749e7ddc/lib/linguist/documentation.yml#L1-L2

To fix the issue try adding the line `docs/ linguist-documentation=false` to file `.gitattributes`. (If that doesn't work, add `**/*` right after `docs/`.)

**Reply by gurjeetsinghvirdee** [reply]
I have the same issue with this repo
https://github.com/gurjeetsinghvirdee/leetcode-solution-in-js

**Reply by lildude** [reply]
That repo has no code in it. It's only got "prose" in the form of markdown and txt files and binaries in the form of images, none of which count towards the stats as detailed in [How Linguist Works](https://github.com/github/linguist/blob/master/docs/how-linguist-works.md).

Binaries will never count towards the stats but if you want Markdown to, you need to implement an [override](https://github.com/github/linguist/blob/master/docs/overrides.md#using-gitattributes)

**Reply by gurjeetsinghvirdee** [reply]
Thank you
